docs/components/tooltip/test.html

```diff
@@ -1,16 +1,12 @@
 ---
-title: Testing - Tooltips
+title: Testing - Tooltip
+crumbs:
+  - path: components/tooltip
+    label: Tooltip
 ---
 {% extends 'test.njk' %}
-{% block content %}
-  <section>
-    <nav class="hxBreadcrumb">
-      <a href="components/tooltips">Tooltips</a>
-      <hx-icon class="delimiter" type="angle-right"></hx-icon>
-      <a href="#">{{page.title}}</a>
-    </nav>
-  </section>
 
+{% block content %}
```

Comment:
We're missing section `id`s for the **side panel**, though not a major issue.

Reply:
yup, that'll be handled in a future PR, when we normalize the test pages in preparation for automated testing